ssh-key/Cargo.toml

```diff
@@ -84,6 +84,7 @@ p384 = ["dep:p384", "ecdsa"]
 p521 = ["dep:p521", "ecdsa"]
 ppk = ["dep:hex", "alloc", "cipher/aes-cbc", "dep:hmac", "dep:argon2", "dep:sha1"]
 rsa = ["dep:bigint", "dep:rsa", "alloc", "rand_core"]
+sha1 = ["dep:sha1"]
```

Comment:
This is actually implicitly defined
https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies

Reply:
It's used in the `ppk` feature

>  If you specify the optional dependency with the dep: prefix anywhere in the [features] table, that disables the implicit feature.

Reply:
oh, missed that. Thanks